rendering/fbx.ksy

```diff
@@ -0,0 +1,167 @@
+meta:
+  id: kaydara_fbx
+  title: Kaydara FBX (Filmbox) 3D model
+  file-extension: fbx
+  application: Autodesk/Kaydara Filmbox
+  encoding: UTF-8
+  endian: le
+  license: MIT
+
+doc: |
+  A proprietary file format developed by Kaydara and owned by Autodesk
+  since 2006. It is used to provide interoperability between digital content
+  creation applications. FBX is also part of Autodesk Gameware, a series of
+  video game middleware.
+
+seq:
+  - id: header
+    type: header
+  - id: records
+    type: node_record
+    repeat: until
+    repeat-until: _.end_offset == 0
+
+types:
+
+  header:
+    seq:
+      - id: magic
+        type: strz
+        valid: '"Kaydara FBX Binary  "'
+        size: 21
+      - id: more_magic
+        valid: '[0x1A, 0x00]'
+        size: 2
+      - id: version
+        type: u4
+
+  node_record:
+    seq:
+      - id: end_offset
+        type: u4
+      - id: property_count
+        type: u4
+      - id: property_list_length
+        type: u4
+      - id: name_length
+        type: u1
+      - id: name
+        type: str
+        size: name_length
+      - id: properties
+        type: property
+        repeat: expr
+        repeat-expr: property_count
+        size: property_list_length
+      - id: children
+        size: end_offset - _root._io.pos
+        type: node_record
+        repeat: until
+        repeat-until: _.end_offset == 0
+        if: not _io.eof
+
+  property:
+    seq:
+      - id: type_code
+        size: 1
+        type: str
+      - id: data
+        type:
+          switch-on: type_code
+          cases:
+            '"I"': s4
+            '"L"': s8
+            '"F"': f4
+            '"D"': f8
+            '"Y"': s2
+            '"C"': u1 # TODO boolean, should probably have a type for it
+            '"i"': array(pseudo_type::s4)
+            '"l"': array(pseudo_type::s8)
+            '"f"': array(pseudo_type::f4)
+            '"d"': array(pseudo_type::f8)
+            '"b"': array(pseudo_type::u1)
+            '"S"': string_data
+            '"R"': raw_binary_data
+
+  string_data:
+    seq:
+      - id: length
+        type: u4
+      - id: value
+        type: str
+        size: length
+  
+  raw_binary_data:
+    seq:
+      - id: length
+        type: u4
+      - id: value
+        size: length
+
+  array:
+    params:
+      # Works around lack of ability to parameterise the actual type by passing a fake one through.
+      # https://github.com/kaitai-io/kaitai_struct/issues/135
```

Comment:
`--affected-by`

Reply:
How does `-affected-by` work, i.e. where exactly can it go? I can't seem find it in the docs.

Reply:
Props starting from `-` are not processed by KSK and are for documentation purposes and for software other than KSC. We would like to have a bot notifying spec authors about the issues affecting their specs being fixed and counting stats on issues to prioritize their fixing. But this bot is not yet implemented.

the proposed annotation format is `-affected-by: <the number of the issue in KS main repo>`. It can also be an array of such numbers.

Reply:
Can it be in any position in the tree then?

Reply:
Likely no. I guess only in dicts that are not expected to have user-supplied identifiers as keys.